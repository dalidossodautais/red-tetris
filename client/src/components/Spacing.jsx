import styled from "styled-components";

export default styled.div`
  height: ${(props) => props.height};
  min-height: ${(props) => props.minHeight};
  width: ${(props) => props.width};
`;
