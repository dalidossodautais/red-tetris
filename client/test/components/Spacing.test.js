import React from "react";
import renderer from "react-test-renderer";
import Spacing from "../../src/components/Spacing";

describe("Spacing", () => {
  test("Standard", () => {
    const component = renderer.create(<Spacing />);
    // rendering
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    // re-rendering
    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
