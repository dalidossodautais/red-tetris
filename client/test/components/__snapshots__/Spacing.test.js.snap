// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacing Standard 1`] = `
<div
  className="sc-bdfBwQ gWRubP"
/>
`;

exports[`Spacing Standard 2`] = `
<div
  className="sc-bdfBwQ gWRubP"
/>
`;
